src/components/Comparison.jsx

```diff
@@ -21,7 +22,7 @@ const Comparison = ({
                 className={`ic-icon-with-sub ${menuOpen ? 'active' : ''}`}
                 tooltip-title={t.translate('Comparison')}
             />
-            {activeComparisonsNo ? <span className={`notification-badget ${activeComparisonsNo > 9 ? 'x2' : ''}`}>{activeComparisonsNo}</span> : ''}
+            <NotificationBadge badge={activeComparisonsNo} />
```

Comment:
consider renaming the prop `badge` to `notificationCount` or something more reflective of what it does

Reply:
@brucebinary think the `<MenuBadge  />` should be better

Reply:
@shayanbinary I'm talking about the prop name not the component name. Component name is fine.

Reply:
@bruceoutdoors  does component name is ok? I think Notification is the different object in the app.

Reply:
@shayanbinary no. At least do google on the words "notification badge" and "menu badge" and see what they return.

Reply:
@brucebinary  You are right, I keep this in mind.